exercises/tournament/description.md

```diff
@@ -60,8 +60,3 @@ Devastating Donkeys;Courageous Californians;draw
 ```
 
 Means that the Devastating Donkeys and Courageous Californians tied.
-
-Your program should only process input lines that follow this format.
```

Comment:
> > Wouldn't it be the case that forcing the student to look at every test case to answer that question is going against the one-test-at-a-time metholodogy?
>
> Why would they be writing code to handle it without a failing test first?

I don't think someone doing test-first will write that code without a test. But I don't think this affects whether I would want this question answered (whether I want the sentence to appear in the README) before seeing/writing a test. 

If I pretend that I'm writing tests from scratch, I imagine I am aware of some set of requirements before having written any test and they inform the tests that I will write. I would look to the README for those requirements.

The general question would be stated as "What is the role of an exercise's README? To what extent need it describe requirements and non-requirements?" I would say that https://github.com/exercism/xgo/issues/189 affects how I think about this question (steers me to ask more of a README).

Maybe nobody will worry about this. After all, the proposed sentence states a non-requirement. The consequence of omission is that maybe people code too defensively. That's less severe than the consequence of omitting a requirement.

It also does not really affect me since I generally do look at all the tests beforehand.

> > "should I expect invalid lines?"
>
> Isn't "invalid input will not be tested" up to the tracks to decide, since they are free to add or remove tests.

Oh! So benefit of leaving the sentence off is to give tracks the option. Okay.

I hope we don't run into a situation where all tracks that don't take the option have to each add a HINTS.md saying "we won't test invalid inputs" if we leave the option open **and** students express a preference for answering the question upfront in the README. Of course, we don't need to think about that until it becomes a problem.

---

As I hope is indicated by my prior approval, it does not matter to me whether the proposed sentence is added or not, but I'm happy to continue discussing if it inspires anyone else to have a strong opinion.